index.js

```diff
@@ -104,6 +104,11 @@ function FSWatcher(_opts) {
       opts.usePolling = !!envLower
     }
   }
+  var envInterval = process.env.CHOKIDAR_INTERVAL;
+  if (envInterval !== undefined) {
+    var envInteger = parseInt(envInterval);
+    opts.interval = envInteger;
```

Comment:
Combining these two lines to just `opts.interval = parseInt(envInterval)` should be fine.